Inc/constants.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * constants.h
+ *
+ *  Created on: 18 de mar de 2022
+ *      Author: Thiago
+ */
+
+#ifndef CONSTANTS_H_
+#define CONSTANTS_H_
+
+//GENERAL DEFINES
+#define NUMBER_OF_PACKS				1//6
+#define NUMBER_OF_SLAVES			1//9
+#define NUMBER_OF_CELLS				12
+#define NUMBER_OF_THERMISTORS		5
+#define NUMBER_OF_PACKS_IN_SERIES	1//2
+
+//BATTERY CELL DEFINES
+#define VOLTAGE_MAX_DISCHARGE		34000
+#define VOLTAGE_MAX_CHARGE			35000
+#define VOLTAGE_MAX_BALANCE			34000
+
+#define VOLTAGE_MIN_DISCHARGE		27000
+#define VOLTAGE_MIN_CHARGE			23000
+#define VOLTAGE_MIN_BALANCE			23000
+
+#define TEMPERATURE_MAX_DISCHARGE	550
+#define TEMPERATURE_MAX_CHARGE		500
+#define TEMPERATURE_MAX_BALANCE		400
+
+//CELL BALANCE DEFINES
+#define BALANCE_THRESHOLD_VOLTAGE	10
+#define TIME_BALANCING_SEC			30
+#define DELAY_AFTER_BALANCE_SEC		3
+
+//ADC DEFINES
+#define NUMBER_OF_ADC_CHANNELS 	NUMBER_OF_CURRENT_SENSORS*CHANELLS_PER_CURRENT_SENSOR
```

Comment:
quando fizer defines com operações coloque sempre entre parenteses. Em algumas situações isso pode da problema pq um comando só da efeito num membro da operação, e não no define todo.
```suggestion
#define NUMBER_OF_ADC_CHANNELS 	(NUMBER_OF_CURRENT_SENSORS*CHANELLS_PER_CURRENT_SENSOR)
```